zerver/views/users.py

```diff
@@ -500,9 +500,7 @@ def patch_bot_backend(
         check_valid_interface_type(service_interface)
         assert service_interface is not None
         do_update_outgoing_webhook_service(
-            bot,
-            interface=service_interface,
-            base_url=service_payload_url,
+            bot, interface=service_interface, base_url=service_payload_url, acting_user=user_profile
```

Comment:
You can prevent this change in line-wrapping by putting a `,` after the last option.